### ISSUE TITLE
Implement Functionality to Avoid Unwanted Shadows

### ISSUE THREAD

**Issue by Laserwolve** [Laserwolve-Games/DazStudioScripts#21]
Consider the following image:
![008](https://user-images.githubusercontent.com/87336074/236319595-d5c49f38-32b4-469d-a4cd-231c6aa6cbec.png)
It's the 9th frame of the shadow of the `defaultChest` equipment for the `man` character, at -45 degrees for the block animation. It's comprised of a single black pixel.

This is intentional. However, most of the frames in this particular animation are completely blank. The ones that aren't, look like the above.

In conclusion, we can probably wholly exclude `man_defaultChest_shadow`, and any other shadows that are for gear that is form-fitting enough to render the resultant shadow as virtually non-existent.

We'll probably need to add an `unwantedShadows.csv` file, just like we did with unwanted gear/animation combos.

**Comment by Laserwolve** [comment]
Turns out our alpha clamping was actually completely eradicating lighter shadows, especially ones close to the body. We've removed alpha clamping and figured out a way to get shadows how we like them within Construct 3.

Depending on how many completely blank images we have going forward, we still may need to implement the shadow cropping batch file currently in `archive`.
